modules/api/php/endpoints/candidate/visit/instrument/instrument.class.inc

```diff
@@ -190,6 +190,8 @@ class Instrument extends Endpoint implements \LORIS\Middleware\ETagCalculator
             );
         }
 
+        $data['Data']['candID'] = $data['Meta']['Candidate'];
```

Comment:
I don't understand what you're trying to do.

The API has tests which are currently passing. Why are they passing if it's giving a 500 error? The tests include checking the status code:

https://github.com/aces/Loris/blob/main/raisinbread/test/api/LorisApiInstruments_v0_0_3_Test.php#L136-L155

https://github.com/aces/Loris/blob/main/raisinbread/test/api/LorisApiInstruments_v0_0_3_Test.php#L163-L183

The value you're setting is just passed to _saveValues below. _saveValues then doesn't do anything with it except for unset it, with a comment saying "do not alter this when overwriting the method"

https://github.com/aces/Loris/blob/main/php/libraries/NDB_BVL_Instrument.class.inc#L785-L793